### PR TITLE
Remove verified_gross_mass field

### PR DESCRIPTION
**Pull request by HenrikHL**
If decided by MSC...

### CODE REVIEW

**Review by nt-as — COMMENTED** [review]
Requires changes to EBL to remove the usage of the field.

**Review by nt-as — CHANGES_REQUESTED** [review]
The field is referenced in 07 test data sql:

```

INSERT INTO dcsa_im_v3_0.shipment_equipment (
    id,
    shipment_id,
    equipment_reference,
    verified_gross_mass,
    cargo_gross_weight,
    cargo_gross_weight_unit
) VALUES (
    uuid('e0b81540-4066-11eb-9a35-7734806583a6'),
    uuid('561a5606-402e-11eb-b19a-0f3aa4962e0f'),
    'BMOU2149612',
    'Verified Gross Mass Dummy text::',
    4000,
    'KGM'
);
```

**Comment by HenrikHL** [comment]
Good spotet - removed :-)

**Review by nt-as — APPROVED** [review]
LGTM

**Comment by HenrikHL** [comment]
I am not sure this is valid any more. VGM should be removed completely (will know by the end of today...)

**Comment by nt-as** [comment]
> 
> 
> I am not sure this is valid any more. VGM should be removed completely (will know by the end of today...)

Well, the title of the PR says "remove" !  :D

File: datamodel/testdata.d/07_test_data_ebl.sql [hunk 1]
```diff
@@ -158,13 +158,15 @@ INSERT INTO dcsa_ebl_v1_0.shipment_equipment (
...
+    verified_gross_mass_unit,
     cargo_gross_weight,
     cargo_gross_weight_unit
 ) VALUES (
     uuid('e0b81540-4066-11eb-9a35-7734806583a6'),
     uuid('561a5606-402e-11eb-b19a-0f3aa4962e0f'),
     'BMOU2149612',
-    'Verified Gross Mass Dummy text::',
+    3330,
+    'KGM'
```
Comment (HenrikHL) [inline datamodel/testdata.d/07_test_data_ebl.sql:169]:
```suggestion
```

File: datamodel/initdb.d/03_dcsa_im_v3_0.sql [hunk 2]
```diff
@@ -247,7 +247,8 @@ CREATE TABLE dcsa_ebl_v1_0.shipment_equipment (
 	id uuid DEFAULT uuid_generate_v4() PRIMARY KEY,
 	shipment_id uuid NOT NULL REFERENCES dcsa_ebl_v1_0.shipment (id),
 	equipment_reference varchar(15) NOT NULL REFERENCES dcsa_ebl_v1_0.equipment (equipment_reference),
-	verified_gross_mass varchar(250) NULL,
+	verified_gross_mass real NULL,
+	verified_gross_mass_unit varchar(3) NULL,
```
Comment (HenrikHL) [inline datamodel/initdb.d/03_dcsa_im_v3_0.sql:251]:
```suggestion
```

File: datamodel/initdb.d/03_dcsa_im_v3_0.sql [hunk 3]
```diff
@@ -247,7 +247,8 @@ CREATE TABLE dcsa_ebl_v1_0.shipment_equipment (
 	id uuid DEFAULT uuid_generate_v4() PRIMARY KEY,
 	shipment_id uuid NOT NULL REFERENCES dcsa_ebl_v1_0.shipment (id),
 	equipment_reference varchar(15) NOT NULL REFERENCES dcsa_ebl_v1_0.equipment (equipment_reference),
-	verified_gross_mass varchar(250) NULL,
```
Comment (HenrikHL) [inline datamodel/initdb.d/03_dcsa_im_v3_0.sql:250]:
```suggestion
```

File: datamodel/initdb.d/03_dcsa_im_v3_0.sql (hunk 2 above)
Comment (HenrikHL) [inline datamodel/initdb.d/03_dcsa_im_v3_0.sql:251]:
```suggestion
```

**Comment by HenrikHL** [comment]
It has been agreed that the VGM should be removed

**Review by nt-as — APPROVED** [review]

**Pull request merged**